### ISSUE TITLE
Steps for Execution

### ISSUE THREAD

**Issue by YiwenWu** [deepnarainsingh/RiseInPrice#1]
the steps is too simple , i can not understand it , do you have more detailed steps? thank you so much

**Comment by deepnarainsingh** [comment]
I agree the steps are not in too detail because of the time constraints.
can you let me know where you are facing the problem so that i can help you. i have plan to improve the instruction details but not getting time.

**Comment by YiwenWu** [comment]
like the version of each software, like the version of  python, spark, kafka....
i can't install the pyspark and happybase. do you have any manual. thank you so much

**Comment by deepnarainsingh** [comment]
yeah nice suggestions. I am tight on timelines , please drop me a email and i will send you the details.

**Comment by YiwenWu** [comment]
thank you so much!!, 492946032@qq.com

**Comment by xixiongfen** [comment]
@deepnarainsingh ,I am reading your program,can u provide detailed material? 
my mail:xixiongfen@bupt.edu.cn,    thank  u!!!

**Comment by deepnarainsingh** [comment]
@xixiongfen Hi I can provide you the details after 10th may, i am currently busy in my academic project.
Appreciate your patience. Also drop me a email with list of details which you are looking for.

**Comment by xixiongfen** [comment]
@deepnarainsingh I have build the requirements its need,but cann't run. can u provide the details about uberAPI，crontab and webUI ,.In addition, u say"Historical analysis on the price surge data to study the surge behavior and how it relates with time.",but I cann't find the process of historical analysis
.  my mail is:    xixiongfen@bupt.edu.cn, thank u!!!

**Comment by deepnarainsingh** [comment]
Hi

U will have to set up the cronjob (uber_job.cron)which is there in the repo. Change the path mentioned in file as per your setting.
You can setup cronjob in linux machines and not on Mac OS. They have something different which I am not sure of. Just check on that part.

You need to install uber_rides package for Uber API call.
For the UI you will have to install apache flask.

So Historical analysis is done when real time data is processed , the flag is changed and which is used for doing the historical analysis and it is displayed on UI.

If you are documenting the steps for setting up please contribute . I will mention your name in credits.

Please let me know if you face any issue . Happy to help.

Thanks,
Deep

**Comment by deepnarainsingh** [comment]
For setting up the cronjob on linux 

got to command prompt type the command 

crontab  /"Path to the file"/uber_job.cron

**Comment by vibhortaneja** [comment]
Hey, can you mail me (vtaneja127@gmail.com) the detailed report, as I am planning to do something similar for my academic project! Would be grateful , thanks in advance.

**Comment by deepnarainsingh** [comment]
Hi Vibhor,

I have not prepared any report as such. You can read the readme instructions and in case of any issues please let me know , I may help you.

**Comment by vibhortaneja** [comment]
I tried executing, according to your steps mentioned, I am facing several issues, can you mail me detailed steps of execution and integration. Would be grateful , thanks in advance.

**Comment by Denver624** [comment]
Hi Deep,
Can you tell me how to set up the table in Hbase and how to create topics in Kafka?
Thank you
